sixtyfps_compiler/expression_tree.rs

```diff
@@ -976,11 +1012,22 @@ impl Expression {
         }
     }
 
+    /// Try to mark this expression to a lvalue that can be assigned to.
+    ///
     /// Return true if the expression is a "lvalue" that can be used as the left hand side of a `=` or `+=` or similar
-    pub fn is_rw(&self) -> bool {
+    pub fn try_set_rw(&mut self) -> bool {
```

Comment:
Not in scope of this request: I find `_rw` a bit of a cryptic suffix :) . Maybe this could be called `try_mark_for_assignment` ?